checker/tests/nullness/flow/Issue1214.java

```diff
@@ -0,0 +1,126 @@
+// Test case for issue #1214:
+// https://github.com/typetools/checker-framework/issues/1214
+
+public class Issue1214 {
+    static String ng1() {
+        String s = "not null";
+        try {
+            int data = 50 / 0;
```

Comment:
All tests are with the literal `0`. What happens if the 0 is in a variable? What about:
```
static String ng2(int x) {
    String s = "not null";
    try {
        short data = (short) (50 / x);
     } catch (Exception e) {
        try {
            s = null;
        } catch (Exception ee) {
        }
    }
    return s;
}
```